cmd/frontend/graphqlbackend/git_commit.go

```diff
@@ -68,7 +68,16 @@ type GitCommitResolver struct {
 // commit will be loaded lazily as needed by the resolver. Pass in a commit when
 // you have batch-loaded a bunch of them and already have them at hand.
 func NewGitCommitResolver(db database.DB, gsClient gitserver.Client, repo *RepositoryResolver, id api.CommitID, commit *gitdomain.Commit) *GitCommitResolver {
+	var repoName api.RepoName
+	if repo != nil {
+		repoName = repo.RepoName()
+	}
+
 	return &GitCommitResolver{
+		logger: log.Scoped("gitCommitResolver", "resolve a specific commit").
+			With(log.Object("commit",
+				log.String("repo", string(repoName)),
+				log.String("commitId", string(id)))),
```

Comment:
Nit: Suggested indentation

```suggestion
		logger: log.Scoped("gitCommitResolver", "resolve a specific commit").With(
			log.Object("commit",
				log.String("repo", string(repoName)),
				log.String("commitId", string(id))
			),
		),
```

This way we make it visually easier to identify where the `log.Object` starts and finishes. 

Although with the earlier suggestion to remove the `log.Object` this suggestion is outdated, but I wrote this as review for next time you encounter a similar indentation challenge. 😀

Reply:
updated